src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -122,6 +125,7 @@
     /** Length of time, in seconds, that {@link #disabled} should auto-disable for if we encounter an error. */
     private @CheckForNull Integer errorDuration;
 
+
```

Comment:
I'm not keen on unnecessary blank lines, or unnecessary changes to whitespace.
Please undo this line.